### PR TITLE
[MOB-1192] Remove animation from Dialog Fragment startup

### PR DESCRIPTION
**Pull request by Ayyanchira**
Hiding and showing in-app before resizing to avoid animation effect.

### CODE REVIEW

**Comment by Ayyanchira** [comment]
I am assuming resize script is called every time for all inapps. Tried on full screen inapps and center aligned inapps

**Comment by Ayyanchira** [comment]
> What happens if it takes time to load? Is it possible to interact with the app while the in-app is on the screen but invisible?

back button works and can still take the user to previous screen. Will have to check what if the app itself is fullscreen and there is no back button in the context.

**Comment by Ayyanchira** [comment]
And also we are showing the window after the resize script is run. So before I check, I assume it is not dependent on the actual data getting loaded. Will check it with the image delay link that you used in one of the in-apps before...

**Comment by vbabenkoru** [comment]
Resize is only called after the entire HTML content is loaded. If there is any asset that takes a long time to load, it will delay the `resize` call.

**Comment by Ayyanchira** [comment]
Oh yes... Thats true...

**Comment by Ayyanchira** [comment]
Oh... one observation:
Even if we use deelayme.com to try delaying the image resource inside the InApps html, onPageFinished is called even before the image is loaded. I think its because the html by itself is loaded from disk and not from web directly. May be thats why on page finished is getting called even before the resources inside aren't completely loaded?

**Comment by Ayyanchira** [comment]
> What happens if it takes time to load? Is it possible to interact with the app while the in-app is on the screen but invisible?

Checked with setting the delay load images of 10 seconds in inapp.. 
Not only we can interact with app while the inapp is invisible, but it exceeded the expectation by showing the inapp 10 seconds later even if we hit back button and switch to different activity altogether (which again is not what we wanted) 😅

**Comment by Ayyanchira** [comment]
Rebased with master

File: iterableapi/src/main/java/com/iterable/iterableapi/IterableInAppFragmentHTMLNotification.java [hunk 1]
```diff
@@ -170,16 +157,13 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
         if (orientationListener == null) {
             orientationListener = new OrientationEventListener(getContext(), SensorManager.SENSOR_DELAY_NORMAL) {
                 public void onOrientationChanged(int orientation) {
-                    // Resize the webview on device rotation
-                    if (loaded) {
```
Comment (vbabenkoru) [inline iterableapi/src/main/java/com/iterable/iterableapi/IterableInAppFragmentHTMLNotification.java:174]: Do we not need this anymore?

File: iterableapi/src/main/java/com/iterable/iterableapi/IterableWebViewClient.java [hunk 2]
```diff
@@ -27,19 +28,22 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
         return true;
     }
 
+    @Override
+    public void onPageStarted(WebView view, String url, Bitmap favicon) {
```
Comment (vbabenkoru) [inline iterableapi/src/main/java/com/iterable/iterableapi/IterableWebViewClient.java:32]: Looks like this is not doing anything.

File: iterableapi/src/main/java/com/iterable/iterableapi/IterableInAppFragmentHTMLNotification.java (hunk 1 above)
Comment (Ayyanchira) [reply iterableapi/src/main/java/com/iterable/iterableapi/IterableInAppFragmentHTMLNotification.java:174]: this was a boolean flag which was replaced with tri-state : not loaded, initialized and finished loading, which was then replaced by our new granular refresh.

File: iterableapi/src/main/java/com/iterable/iterableapi/IterableWebViewClient.java (hunk 2 above)
Comment (Ayyanchira) [reply iterableapi/src/main/java/com/iterable/iterableapi/IterableWebViewClient.java]: Thanks...
Removed... 👍

File: iterableapi/src/main/java/com/iterable/iterableapi/IterableInAppFragmentHTMLNotification.java (hunk 1 above)
Comment (vbabenkoru) [reply iterableapi/src/main/java/com/iterable/iterableapi/IterableInAppFragmentHTMLNotification.java:174]: Hmm, but this was there before.. Was there a reason why we can't resize on rotation until it's loaded? Seems too specific.

**Pull request merged**